packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -34,13 +93,27 @@ export default {
 @use '@justeat/fozzie/src/scss/fozzie' as f;
 
 .c-rating {
-    display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid red;
-    font-family: f.$font-family-base;
-    @include f.font-size(heading-m);
+    .c-rating-star-wrapper {
+        margin: 0;
+        padding: 0;
+        list-style-type: none;
+
+        .c-rating-star {
+            display: inline-block;
+            width: 15px; // Todo - decide on how to size these. Will create a ticket around this.
+        }
+
+        .c-rating-star-filled {
+            & path {
+                fill: f.$color-support-brand-01;
+            }
+        }
+
+        .c-rating-star-empty {
+            & path {
+                fill: f.$color-mozzarella-50;
+            }
+        }
+    }
```

Comment:
Because of the naming scheme, you shouldn't need to nest these inside `c-rating` (just have it outside so the selector specificity is as low as possible)